dcpy/lifecycle/ingest/extract.py

```diff
@@ -35,11 +36,10 @@ def download_file_from_source(
         case S3Source():
             s3.download_file(source.bucket, source.key, path)
         case ScriptSource():
```

Comment:
This still isn't actually used yet at all, so it's a little weird at the moment. Will likely be renamed/refactored at some point (or removed entirely)